frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
+}
+
+.products-component {
+  margin: 20px 5px 20px 5px;
+  padding: 5px 5px 5px 5px;
+
+  display: inline-block;
+
+  //background-color: #325774;
+  //below is site color, i prefer the above though
+  background-color: #363f46;
```

Comment:
you can define colors in the `_colors.scss` file as variables with a name, and you can use those variables so you don't have hex colors lying around